frontend/src/metabase/query_builder/components/template_tags/CardTagEditor.jsx

```diff
@@ -78,7 +78,9 @@ export default class CardTagEditor extends Component {
           {cardId == null ? (
             t`Question #…`
           ) : (
-            <Link to={questionUrl(cardId)}>{t`Question #${cardId}`}</Link>
+            <Link
+              to={Urls.question(question ? question : { id: cardId })}
+            >{t`Question #${cardId}`}</Link>
```

Comment:
Wdyt of definining a function for this above `render`:

```js
  buildLinkToQuestion(question, cardId) {
    return Urls.question(question || { id: cardId });
  }
```

and then making the Link call a one-liner by calling the function:

```suggestion
         <Link to={this.buildLinkToQuestion(question, cardId)}>
              {t`Question #${cardId}`}
         </Link>
```

Reply:
…maybe not even a function, just a constant set outside render may do the trick.

Reply:
Good point, thank you! Extracted a method for now. Defining a constant can be a bit confusing because there are cases when both `question` and `cardId` are undefined (during loading). That's why there is a `cardId == null` ternary above